sensorimotor/experiments/capacity/experiment.py

```diff
@@ -47,168 +47,233 @@
 
 from nupic.research.monitor_mixin.monitor_mixin_base import MonitorMixinBase
 
+from sensorimotor.exhaustive_one_d_agent import ExhaustiveOneDAgent
 from sensorimotor.one_d_world import OneDWorld
 from sensorimotor.one_d_universe import OneDUniverse
 from sensorimotor.random_one_d_agent import RandomOneDAgent
-from sensorimotor.exhaustive_one_d_agent import ExhaustiveOneDAgent
-
 from sensorimotor.sensorimotor_experiment_runner import (
   SensorimotorExperimentRunner
 )
 
-
-
-# Constants
+# Default number of input bits
 N = 512
-W = 20
-DEFAULTS = {
-  "n": N,
-  "w": W,
-  "tmParams": {
-    "columnDimensions": [N],
-    "minThreshold": W*2,
-    "activationThreshold": W*2,
-    "maxNewSynapseCount": W*2
-  },
-  "tpParams": {
-    "columnDimensions": [N],
-    "numActiveColumnsPerInhArea": W,
-    "potentialPct": 0.9,
-    "initConnectedPct": 0.5
-  }
-}
-VERBOSITY = 1
-PLOT = 1
-SHOW_PROGRESS_INTERVAL = 10
-
-
-
-def run(numWorlds, numElements, outputDir, params=DEFAULTS):
-  # Extract params
-  n = params["n"]
-  w = params["w"]
-  tmParams = params["tmParams"]
-  tpParams = params["tpParams"]
-
-  # Initialize output
-  if not os.path.exists(outputDir):
-    os.makedirs(outputDir)
-
-  csvFilePath = os.path.join(outputDir, "{0}x{1}.csv".format(numWorlds,
-                                                             numElements))
 
```

Comment:
This one: https://github.com/numenta/nupic.research/pull/135/files#diff-3471a27934a13cf92b8c665a86d6e504R60